src/models/multiChart.js

```diff
@@ -213,6 +213,9 @@ nv.models.multiChart = function() {
                 return a.map(function(aVal,i){return {x: aVal.x, y: aVal.y + b[i].y}})
             }).concat([{x:0, y:0}]) : [];
 
+            x = lines1.xScale();
```

Comment:
This is my try on solving this issue (#2039)
Its really simple, just add a new option xScale on your chart (like the lineChart)